Makefile.in

```diff
@@ -225,6 +225,11 @@ uninstall.mcode.program:
 
 install.mcode: install.mcode.program install.vhdllib
 
+install.mcode.dll_deps:
```

Comment:
Don't you need to call this target somewhere?

Dependencies of DLLs can be recursive. See here:  
![image](https://user-images.githubusercontent.com/956109/212464017-2c2c1ea4-dc7b-4aa7-97fc-b626dbbd6986.png)

I used https://github.com/lucasg/Dependencies to find dependencies on Windows.